### PR TITLE
docs(readme): add hours, mult, omm

### PR DESCRIPTION
**Pull request by dhth**
Adds the following:

- [hours](https://github.com/dhth/hours) - is a no-frills time tracking toolkit for the command line
- [mult](https://github.com/dhth/mult) - lets you run a command multiple times and glance at the outputs via a TUI
- [omm](https://github.com/dhth/omm) - is a keyboard-driven task manager for the command line

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,10 +81,12 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
 - [mergestat](https://github.com/mergestat/mergestat) - Run SQL queries on git repositories.
 - [meteor](https://github.com/stefanlogue/meteor) - A highly customizable conventional commit message tool.
 - [mods](https://github.com/charmbracelet/mods) - AI on the CLI, built for pipelines.
+- [mult](https://github.com/dhth/mult) - lets you run a command multiple times and glance at the outputs via a TUI
```
Comment (Delta456) [inline README.md:84]: Add punctuation at the end

File: README.md [hunk 2]
```diff
@@ -80,10 +81,12 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
...
 - [nachrichten](https://github.com/zMoooooritz/nachrichten) - Access up-to-date news in German provided by the [Tagesschau](https://www.tagesschau.de/).
 - [Neon Modem Overdrive](https://github.com/mrusme/neonmodem) - A BBS-style TUI client for Discourse, Lemmy, Lobste.rs and Hacker News.
 - [nom](https://github.com/guyfedwards/nom) - An RSS reader and manager.
 - [Noted](https://github.com/torbratsberg/noted) - A note viewer and manager.
+- [omm](https://github.com/dhth/omm) - is a keyboard-driven task manager for the command line
```
Comment (Delta456) [inline README.md:89]: Same here

**Review by Delta456 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dhth) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -80,10 +81,12 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
 - [mergestat](https://github.com/mergestat/mergestat) - Run SQL queries on git repositories.
 - [meteor](https://github.com/stefanlogue/meteor) - A highly customizable conventional commit message tool.
 - [mods](https://github.com/charmbracelet/mods) - AI on the CLI, built for pipelines.
+- [mult](https://github.com/dhth/mult) - lets you run a command multiple times and glance at the outputs via a TUI.
```
Comment (Delta456) [inline README.md:84]:
```suggestion
- [mult](https://github.com/dhth/mult) - Run a command multiple times and glance at the outputs via a TUI.
```

File: README.md [hunk 4]
```diff
@@ -80,10 +81,12 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
...
 - [nachrichten](https://github.com/zMoooooritz/nachrichten) - Access up-to-date news in German provided by the [Tagesschau](https://www.tagesschau.de/).
 - [Neon Modem Overdrive](https://github.com/mrusme/neonmodem) - A BBS-style TUI client for Discourse, Lemmy, Lobste.rs and Hacker News.
 - [nom](https://github.com/guyfedwards/nom) - An RSS reader and manager.
 - [Noted](https://github.com/torbratsberg/noted) - A note viewer and manager.
+- [omm](https://github.com/dhth/omm) - is a keyboard-driven task manager for the command line.
```
Comment (Delta456) [inline README.md:89]:
```suggestion
- [omm](https://github.com/dhth/omm) - A keyboard-driven task manager for the command line.
```

File: README.md [hunk 5]
```diff
@@ -69,6 +69,7 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
 - [got](https://github.com/fedeztk/got) - A simple translator and text-to-speech app built on simplytranslate's APIs.
 - [gum](https://github.com/charmbracelet/gum) - Interactivity and styling for shells and shell scripts.
 - [hiSHtory](https://github.com/ddworken/hishtory) - Your shell history in context: synced, and queryable.
+- [hours](https://github.com/dhth/hours) - is a no-frills time tracking toolkit for the command line.
```
Comment (Delta456) [inline README.md:72]:
```suggestion
- [hours](https://github.com/dhth/hours) - A no-frills time tracking toolkit for the command line.
```

File: README.md (hunk 3 above)
Comment (dhth) [reply README.md]: Ah, didn't see the latest build results. Changes pushed.

**Review by Delta456 — APPROVED** [review]

**Pull request merged**